webapp/locales/en.json

```diff
@@ -317,7 +317,9 @@
         "their-perspective": "Vice versa: The blocked person will also no longer see your posts in their news feed.",
         "search": "Posts of blocked people disappear from your search results.",
         "notifications": "Blocked users will no longer receive notifications if they are mentioned in your posts.",
-        "closing": "This should be sufficient for now so that blocked users can no longer bother you."
+        "closing": "This should be sufficient for now so that blocked users can no longer bother you.",
+        "commenting-disabled": "Commenting is not possible at this time on this post.",
+        "commenting-explanation": "This can happen for several reasons, please see "
```

Comment:
That's the thing. It can happen only for one reason at the moment: You got blocked. If this is indeed the case, I would be transparent about this. Exposing this information is exactly what we discussed with @DennisHack and it didn't seem to a problem.

Reply:
We have talked about following some guidelines and maybe using Mastodon as a starting point.
This post explains that blocked users are not notified in any way they've been blocked. https://amp.reddit.com/r/Mastodon/comments/a1d5sx/mute_vs_block/

Also, since we have issues open to be able to turn off commenting on Posts, it makes sense to me to either decide we do or don't want to support them in the future. Then, we can either close those issues, or leave a generic message like it is in this PR. Then, we aren't divulging that comments are turned off because the user has been blocked by the other.